src/promptflow-core/promptflow/core/_prompty_utils.py

```diff
@@ -169,6 +169,20 @@ def format_llm_response(response, api, is_first_choice, response_format=None, st
     """
     Format LLM response
 
+    If is_first_choice is false, it will directly return LLM response.
```

Comment:
add a line saying:
if is_first_choice is true, behavior as blow:

Reply:
fix it